prime-router/src/main/kotlin/fhirengine/utils/HL7MessageHelpers.kt

```diff
@@ -63,7 +63,9 @@ object HL7MessageHelpers : Logging {
                 null,
                 null,
                 null,
-                "0" // Hash is only used for deduplication when receiving
+                "0", // Hash is only used for deduplication when receiving
+                sourceReportId,
```

Comment:
Same question here as above

Reply:
Yeah, these need to be discussed on a future ticket when metadata is handled for UP